examples/next/pages/_document.js

```diff
@@ -1,3 +1,4 @@
+import React from 'react'
```

Comment:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md

Reply:
Use `overrides` to turn off `react/react-in-jsx-scope` for the Next.js example because Next.js docs don't import React AFAICT.